hibernate-core/src/main/java/org/hibernate/jdbc/Expectations.java

```diff
@@ -80,11 +80,11 @@ else if ( rowCount == -3 ) {
 			}
 		}
 
-		private void checkNonBatched(int rowCount, PreparedStatement statement) {
+		private void checkNonBatched(int rowCount, PreparedStatement statement, String statementSQL) {
```

Comment:
just a minor but `PreparedStatement statement` parameter can be removed, the same for the `checkBatched` method

Reply:
Thanks, I'll do that.